api/lib/database/methods.js

```diff
@@ -95,6 +95,93 @@ const getNotifications = ({ hasura_url, hasura_admin_key }) => (
   )
 }
 
+const getNetwork = ({ hasura_url, hasura_admin_key }) => () => async (id) => {
+  return await read(
```

Comment:
here you should return the first result already so you don't need to do it in the rest of the code

Reply:
what do you mean? You mean so I don't need to declare every and each field in the table?

Reply:
see this comment https://github.com/luniehq/lunie/pull/4235#discussion_r444338429